bin/run

```diff
@@ -4,14 +4,12 @@
 use Symfony\Component\Console\Input\ArgvInput;
 use OpenEuropa\TaskRunner\TaskRunner;
 use Symfony\Component\Console\Output\ConsoleOutput;
-use Composer\Autoload\ClassLoader;
 
+$classLoader = null;
```

Comment:
I am afraid this is wrong, the Task Runner constructor expects a ClassLoader object, we cannot pass NULL to it:

```
    public function __construct(InputInterface $input, OutputInterface $output, ClassLoader $classLoader)
```

The fact that we know that, down the road, this is eventually set by Robo is not relevant here, e.g. we should not assume internal Robo behaviors that might change.

Reply:
Oki, i will update accordingly even if i don't fully understand.